SDL/LST.cc

```diff
@@ -1,34 +1,39 @@
 #include "LST.h"
 
-SDL::LST::LST() {
-    TrackLooperDir_ = getenv("LST_BASE");
+namespace {
+  TString trackLooperDir() {
+    return getenv("LST_BASE");
+  }
 }
 
 void SDL::LST::eventSetup() {
     static std::once_flag mapsLoaded;
-    std::call_once(mapsLoaded, &SDL::LST::loadMaps, this);
+    std::call_once(mapsLoaded, &SDL::LST::loadMaps);
+
     TString path = get_absolute_path_after_check_file_exists(
-        TString::Format("%s/data/centroid_CMSSW_12_2_0_pre2.txt",TrackLooperDir_.Data()).Data());
+        TString::Format("%s/data/centroid_CMSSW_12_2_0_pre2.txt",trackLooperDir().Data()).Data());
     static std::once_flag modulesInited;
     std::call_once(modulesInited, SDL::initModules, path);
 }
```

Comment:
Should this be deleted, now that the maps are loaded in the ESProducer? Or has it stayed until we know that the ESRpoducer method works and will be deleted in the future?